.changes/unreleased/tinymce-TINY-11661-2024-12-23.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Not specifying a notification type fallback to 'info'
```

This will need to be noted in the QA and Docs section of JIRA, too.
```suggestion
body: Not specifying a notification type now defaults to 'info'
```

---

I will note it in the QA and Docs section